integration-cli/docker_cli_daemon_test.go

```diff
@@ -349,4 +351,24 @@ func TestDaemonVolumesBindsRefs(t *testing.T) {
 	}
 
 	logDone("daemon - bind refs in data-containers survive daemon restart")
+
+func TestDaemonKeyGeneration(t *testing.T) {
+	os.Remove("/etc/docker/key.json")
```

Comment:
Probably silly question, but should these paths use getDaemonConfDir(). Probably moot, 'cause there's no Windows daemon (yet)